embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/anr/sigquit/SigquitDetectionService.kt

```diff
@@ -51,12 +47,11 @@
 
     private fun setupGoogleAnrTracking() {
         if (configService.anrBehavior.isGoogleAnrCaptureEnabled() && !googleAnrTrackerInstalled.getAndSet(true)) {
-            ThreadUtils.runOnMainThread { setupGoogleAnrHandler() }
+            ThreadUtils.runOnMainThread(logger) { setupGoogleAnrHandler() }
```

Comment:
Worked around the logger being needed in a "static" context by passing it in. We should try to not do this, but to make existing code works, this is fInE in small chunks I guess....